src/main/java/com/icthh/xm/ms/dashboard/web/rest/DefaultProfileResource.java

```diff
@@ -105,6 +108,7 @@ public DefaultProfileResource(DefaultProfileService defaultProfileService,
     @GetMapping("/default-profiles/{id}")
     @Timed
     @PostAuthorize("hasPermission({'returnObject': returnObject.body}, 'DEFAULT_PROFILE.GET_LIST.ITEM')")
+    @PrivilegeDescription("Privilege to get the defaultProfile by id")
```

Comment:
replace defaultProfiles to more informative message. Describe what is defaultProfiles.